lib/API/JSON/v1/Subscribers.php

```diff
@@ -412,6 +418,21 @@ function delete($data = []) {
     }
   }
 
+  function sendConfirmationEmail($data = []) {
```

Comment:
Do you reckon this would deserve a test? There is quite some logic in this method...